icuSources/i18n/CMakeLists.txt

```diff
@@ -13,9 +13,9 @@
 ##===----------------------------------------------------------------------===##
 
 target_include_directories(_FoundationICU PRIVATE .)
-
 target_sources(_FoundationICU
     PRIVATE
+        alphaindex.cpp
```

Comment:
This file doesn't seem to be new. Any idea why this wasn't included in this make file before?

Reply:
I think we might have just forgotten to include it and got lucky with it 😓 
Now these CMakeList files are auto generated so we shouldn't have this problem anymore.